src/lender/assessor.sol

```diff
@@ -276,4 +334,31 @@ contract Assessor is Definitions, Auth, Interest {
 
         return safeSub(ONE, rdiv(seniorAsset, assets));
     }
+
+    // returns the remainingCredit plus a buffer for the interest increase
+    function remainingCredit() public view returns(uint) {
+        if (address(lending) == address(0)) {
```

Comment:
I think we could make a modifier for all lender related methods.

something like:

```javascript
   modifier lenderActive() {
      if (address(lending) != address(0)) {
         _;
      }
      return 0;

   }
```

We would need to check if this works.